File: API/Features/ShipCrews/Dtos/Form/ShipCrewReadDto.cs
using API.Infrastructure.Classes;

namespace API.Features.ShipCrews {

    public class ShipCrewReadDto {

        public int Id { get; set; }
        public SimpleResource Gender { get; set; }
        public SimpleResource Nationality { get; set; }
        public SimpleResource Ship { get; set; }
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public bool IsActive { get; set; }

    }

}

File: FrontEnd/src/app/features/shipCrews/classes/dtos/shipCrew-read-dto.ts
import { GenderActiveVM } from 'src/app/features/genders/classes/view-models/gender-active-vm'
import { NationalityDropdownVM } from 'src/app/features/nationalities/classes/view-models/nationality-dropdown-vm'
import { ShipDropdownVM } from 'src/app/features/ships/classes/view-models/ship-dropdown-vm'

export interface ShipCrewReadDto {

    id: number
    gender: GenderActiveVM
    nationality: NationalityDropdownVM
    ship: ShipDropdownVM
    lastname: string
    firstname: string
    birthdate: string
    isActive: boolean

}
